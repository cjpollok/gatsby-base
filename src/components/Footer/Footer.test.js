import React from 'react'
import { render } from 'enzyme'

import Footer from './Footer'

describe('Footer Component', () => {
  it('renders correctly', () => {
    const wrapper = render(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
