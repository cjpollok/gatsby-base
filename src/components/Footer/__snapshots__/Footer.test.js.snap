// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `
<footer
  class="container"
>
  <div
    class="menu"
  >
    <a
      href="https://github.com/cjpollok/gatsby-base"
      rel="nofollow"
    >
      See on GitHub
    </a>
  </div>
</footer>
`;
